e2e/common/traits/builder_test.go

```diff
@@ -233,6 +234,7 @@ func TestBuilderTrait(t *testing.T) {
 			TestTimeoutShort).Should(ContainSubstring("No such file or directory"))
 
 		Expect(Kamel("delete", "--all", "-n", ns).Execute()).To(Succeed())
+		Expect(Kamel("reset", "-n", ns).Execute()).To(Succeed())
```

Comment:
If we require to reset in the "common" e2e flow, then we should better move this test in the "common-with-custom-install".